hack/update-api-reference-docs.sh

```diff
@@ -37,12 +47,40 @@ echo "Reading swagger spec from: ${SWAGGER_PATH}"
 mkdir -p $V1_PATH
 mkdir -p $V1BETA1_PATH
 
-docker run --rm -v $V1_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v3 \
+docker run -u $(id -u) --rm -v $V1_TEMP_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v4 \
```

Comment:
Thanks @jlowdermilk! So I didn't understand your comments when I reviewed your PR, now it all makes sense!

As you said, the address I passed to -v will be resolved as a host-resolvable path, then we still need to use ${KUBE_TEMP}, there can be race condition if two dockerized tests are running at the same time. Am I right?